src/Storefront/Resources/app/storefront/src/plugin/slider/gallery-slider.plugin.js

```diff
@@ -182,6 +182,35 @@ export default class GallerySliderPlugin extends BaseSliderPlugin {
         this._slider.goTo(currentIndex - 1);
     }
 
+    /**
+     * @private
```

Comment:
I think an explaining comment would be good why we clone the slider controls here.